src/preferences/dlgpreferencepage.h

```diff
@@ -9,6 +11,11 @@ class DlgPreferencePage : public QWidget {
     DlgPreferencePage(QWidget* pParent);
     virtual ~DlgPreferencePage();
 
+    /// Returns the help URL for the current page.
+    /// Subclasses can provide a path to the appropriate manual page by
+    /// overriding this. The default implementation returns an invalid QUrl.
```

Comment:
I'm not a fan of non-functional/empty default implementations. The extraction of `WTrackMenu` has demonstrated why this is a bad idea in terms of maintainability and extensibility. But in this case it is acceptable.